src/resolve-nom.ts

```diff
@@ -32,16 +32,26 @@ export class ResolveNom implements NameResolver {
   }
 
   async resolve(id: string): Promise<NameResolutionResults> {
+    if (!id.endsWith('.nom')) {
+      return {
+        resolutions: [],
+        errors: [],
+      }
+    }
+
+    const name = id.substring(0, id.length - 4)
```

Comment:
This is removing the `.nom`, right? I know it seems obvious but I had to think twice to realize. How do you feel about doing `id.length - '.nom'.length`? Otherwise maybe a comment explaining?